src/rules/no-tags.js

```diff
@@ -0,0 +1,41 @@
+var _ = require('lodash');
+var rule = 'no-tags';
```

Comment:
What about `no-restricted-tags` - this would follow ESLint naming convention for configurable rules:

- no-restricted-properties
- no-restricted-globals
- no-shadow-restricted-names
- no-restricted-modules
- no-restricted-syntax
- no-restricted-imports

http://eslint.org/docs/rules/